src/OrchardCore/OrchardCore.Abstractions/Modules/Services/ILocalClock.cs

```diff
@@ -5,10 +5,14 @@ namespace OrchardCore.Modules;
 /// </summary>
 public interface ILocalClock
 {
+    [Obsolete("This property has been deprecated, please use GetLocalNowAsync() instead.", error: true)]
```

Comment:
```suggestion
    [Obsolete("This property has been deprecated and will be removed in a future version. Please use GetLocalNowAsync() instead.", error: true)]
```

Reply:
Added the same time as https://github.com/OrchardCMS/OrchardCore/pull/16752#discussion_r1765669230.